src/components/AddPlant.js

```diff
@@ -24,31 +24,37 @@ export const AddPlant = () => {
         'Content-Type': 'application/json',
         Authorization: accessToken
       },
-      // eslint-disable-next-line object-shorthand
-      body: JSON.stringify({ plantname: plantname, species: species })
+      body: JSON.stringify({ plantname, species })
     };
     fetch(API_URL('addplant'), options)
       .then((response) => response.json())
       .then((data) => {
         if (data.success) {
-          dispatch(plants.actions.setPlantname(data.response.plantname));
-          dispatch(plants.actions.setSpecies(data.response.species));
-          dispatch(plants.actions.setPlantId(data.response._id))
-          dispatch(plants.actions.setBirthday(data.response.birthday));
-          dispatch(plants.actions.setLastWatered(data.response.lastWatered));
-          dispatch(plants.actions.setLastSoilChange(data.response.lastSoilChange));
+          // creating a new plant-object with data based on the plant-schema:
```

Comment:
great stuff! this also looks like something that would solve the deletePlant not working